src/preset_cli/cli/superset/sync/dbt/datasets.py

```diff
@@ -267,12 +268,34 @@ def compute_columns_metadata(
             dict_merge(final_column, dbt_metadata[column])
             dbt_metadata[column] = final_column
 
+    dbt_calc_columns_ = {c["column_name"]: c for c in dbt_calc_columns}
```

Comment:
Tried using the same var name (`dbt_calc_columns = {c["column_name"]: c for c in dbt_calc_columns}`) but `mypy` was going crazy so had to use this.

Reply:
Yeah, from a pure point of view the type of a variable should never change, and in that case it would change from `list` to `dict`.

Maybe `dbt_calc_columns_by_name`, if it's not too big?

Reply:
great suggestion!